dicom-archive/DICOM/Element.pm

```diff
@@ -272,7 +384,16 @@ sub readLength {
   return ($vrstr, $length);
 }
 
-# Return the values of each field.
+
+=pod
+
+=head3 values()
+
+Returns the values of each field of the object.
+
+RETURNS: values of each field
```

Comment:
Returns the properties of a VR: group, element, offset, code, length, name and value.